std/experimental/allocator/building_blocks/aligned_block_list.d

```diff
@@ -0,0 +1,629 @@
+// Written in the D programming language.
+/**
+Source: $(PHOBOSSRC std/experimental/allocator/building_blocks/_aligned_block_list.d)
+*/
+module std.experimental.allocator.building_blocks.aligned_block_list;
+
+import std.experimental.allocator.common;
+import std.experimental.allocator.building_blocks.null_allocator;
+
+// Common function implementation for thread local and shared AlignedBlockList
+private mixin template AlignedBlockListImpl(bool isShared)
+{
+    import std.traits : hasMember;
+    import std.typecons : Ternary;
+    static if (isShared)
+    import core.internal.spinlock : SpinLock;
+
+private:
+    struct AlignedBlockNode
+    {
+        AlignedBlockNode* next, prev;
+        Allocator bAlloc;
```

Comment:
"b" for block? It would be good if we either rename it to `alloc` or `blockAllocator` to avoid confusion and needless wonderings when looking at the code.